packaging_automation/update_package_properties.py

```diff
@@ -65,12 +69,12 @@ class PackagePropertiesParams:
     fancy_version_number: int
     microsoft_email: str = ""
     name_surname: str = ""
-    latest_changelog: str = ""
     changelog_date: datetime = datetime.now()
+    changelog_entry: str = ""
 
     @property
-    def should_add_changelog(self) -> bool:
-        return not self.fancy or self.fancy_version_number == 1
+    def changelog_version_entry(self):
```

Comment:
```suggestion
    def changelog_version_entry(self) -> str:
```

Reply:
I think you do not always define function return types. You can ignore this suggestion

Reply:
Fixed